02/run.py

```diff
@@ -0,0 +1,37 @@
+with open("input", "r") as file:
+    passwords = file.readlines()
+
+
+def check_password_1(min, max, letter, word):
+    count = list(word).count(letter)
+    return bool(count >= min and count <= max)
```

Comment:
No need for bool here:
```suggestion
    return count >= min and count <= max
```
Or:
```suggestion
    return min <= count <= max
```